### PR TITLE
Remove unused dependency

### PR DESCRIPTION
**Pull request by udda1996**
Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by pubudu91 — CHANGES_REQUESTED** [review]
A couple of points,
- Please pay attention to formatting. There are unnecessary new lines added in files through out the PR.
- Pay attention to commits and commit messages. Most of the commits seem like temporary or duplicate commits made during development. Group the commits appropriately and squash the rest. 
- Look in to rebasing instead of merging when syncing with the upstream branch. That'd help avoid the multiple merge commits scattered throughout the PR. Also would make it easier if/when you want to edit previous commits.
- If removing something, let's remove it instead of commenting it out.

File: build.gradle [hunk 1]
```diff
@@ -20,6 +20,7 @@ allprojects {
             println projectDir
         }
     }
+
```
Comment (keizer619) [inline build.gradle:23]:
```suggestion
```

File: bvm/ballerina-runtime/build.gradle [hunk 2]
```diff
@@ -24,19 +24,13 @@ dependencies {
 //    implementation ('org.apache.ws.commons.axiom:axiom-dom') {
 //        exclude group: 'org.apache.ws.commons.axiom', module: 'axiom-api'
 //    }
-    implementation ('org.apache.ws.commons.axiom:axiom-c14n') {
-        exclude group: 'org.apache.ws.commons.axiom', module: 'axiom-api'
-        exclude group: 'org.apache.ws.commons.axiom', module: 'axiom-impl'
-    }
+
```
Comment (keizer619) [inline bvm/ballerina-runtime/build.gradle:27]:
```suggestion
```

File: cli/central-client/build.gradle [hunk 3]
```diff
@@ -30,7 +30,7 @@ configurations {
 
 dependencies {
     implementation project(':ballerina-runtime')
-    implementation 'javax.ws.rs:javax.ws.rs-api'
+//    implementation 'javax.ws.rs:javax.ws.rs-api'
```
Comment (keizer619) [inline cli/central-client/build.gradle:33]:
```suggestion
```

File: tests/ballerina-test-utils/build.gradle [hunk 4]
```diff
@@ -30,8 +30,8 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3'
     implementation 'org.wso2.transport.http:org.wso2.transport.http.netty'
     implementation 'org.testng:testng'
-    implementation 'org.apache.ws.commons.axiom:axiom-impl'
-    implementation 'org.apache.ws.commons.axiom:axiom-dom'
+//    implementation 'org.apache.ws.commons.axiom:axiom-impl'
+//    implementation 'org.apache.ws.commons.axiom:axiom-dom'
```
Comment (keizer619) [inline tests/ballerina-test-utils/build.gradle:34]:
```suggestion
```

File: tests/jballerina-integration-test/src/test/resources/identifier/ModuleNameClashProject/Dependencies.toml [hunk 5]
```diff
@@ -2,3 +2,5 @@
 org = "ballerina"
 name = "test"
 version = "0.8.0"
+
+
```
Comment (keizer619) [inline tests/jballerina-integration-test/src/test/resources/identifier/ModuleNameClashProject/Dependencies.toml:6]:
```suggestion
```

File: tests/jballerina-integration-test/src/test/resources/observability/metrics_tests/Dependencies.toml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+[[dependency]]
+org = "ballerina"
+name = "testobserve"
+version = "0.0.0"
+
+[[dependency]]
+org = "ballerina"
+name = "observe"
+version = "0.9.0"
+
```
Comment (keizer619) [inline tests/jballerina-integration-test/src/test/resources/observability/metrics_tests/Dependencies.toml:10]:
```suggestion
```

File: tests/jballerina-integration-test/src/test/resources/observability/tracing_tests/Dependencies.toml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+[[dependency]]
+org = "ballerina"
+name = "observe"
+version = "0.9.0"
+
+[[dependency]]
+org = "ballerina"
+name = "testobserve"
+version = "0.0.0"
+
```
Comment (keizer619) [inline tests/jballerina-integration-test/src/test/resources/observability/tracing_tests/Dependencies.toml:10]:
```suggestion
```

File: tests/observability-test-utils/src/test/resources/listener_tests/Dependencies.toml [hunk 8]
```diff
@@ -0,0 +1,6 @@
+[[dependency]]
+org = "ballerina"
+name = "testobserve"
+version = "0.0.0"
+
```
Comment (keizer619) [inline tests/observability-test-utils/src/test/resources/listener_tests/Dependencies.toml:5]:
```suggestion
```

**Pull request closed without merging**